lms/djangoapps/courseware/tests/test_about.py

```diff
@@ -215,7 +215,7 @@ def setUp(self):
         import_course_from_xml(
             self.store,
             'test_user',
-            TEST_DATA_DIR,
+            TEST_DATA_DIR / u'courses_exports',
```

Comment:
How does this work? The `/` operator isn't something I've ever seen applied to strings before.